encryptionkeyrotation/README.md

```diff
@@ -0,0 +1,95 @@
+# CSI Addons Operation: EncryptionKeyRotation
+
+## Terminology
+
+| Term     | Definition                                                                            |
+| -------- | ------------------------------------------------------------------------------------- |
+| VolumeID | The identifier of the volume generated by the plugin.                                 |
+| CO       | Container Orchestration system that communicates with plugins using CSI service RPCs. |
+| SP       | Storage Provider, the vendor of a CSI plugin implementation.                          |
+| RPC      | [Remote Procedure Call](https://en.wikipedia.org/wiki/Remote_procedure_call).         |
+
+## Objective
+
+Define an extension to the CSI specification that will enable storage vendors
+(SP) to develop controllers/plugins that can rotate the Encryption Keys (KEK)
```

Comment:
Where comes the first **K** in KEK come from?

Reply:
Ack, should be Key Encryption Keys 😂